dataplane/linux/int_dataplane.go

```diff
@@ -721,7 +721,9 @@ func (d *InternalDataplane) doStaticDataplaneConfig() {
 	if d.xdpState != nil {
 		if err := d.setXDPFailsafePorts(); err != nil {
 			log.Warnf("failed to set XDP failsafe ports, disabling XDP: %v", err)
-			d.shutdownXDPCompletely()
+			if err := d.shutdownXDPCompletely(); err != nil {
+				log.Panicf("failed to disable XDP: %v", err)
```

Comment:
Don't think we should panic here.  Just log a warn and carry on.  Think this is the case that's tripping people up who have broken kernels.

Reply:
agree . ill update that after the build passes.

Reply:
ill try not to get caught up on what you meant by "broken kernels" :)